.github/workflows/publish.yml

```diff
@@ -5,6 +5,11 @@ on:
     types: [created]
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: npm ci
```

Comment:
This currently only installs dependencies, doesn't it have to build the `.js` files too?

Reply:
`npm run build` is added to the publish spec build job and setup-node action version is incremented
```diff
diff --git a/.github/workflows/publish.yml b/.github/workflows/publish.yml
index dd768c4..84399c8 100644
--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -10,12 +10,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - run: npm ci
+      - run: npm run build
   publish-gpr:
     needs: build
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
           node-version: 20
           registry-url: https://npm.pkg.github.com/
```

Reply:
Not sure if pressing the `[Resolve conversation]` button here is expected or faux pas, so just leaving this open. Feel free to ignore or resolve whichever way.